agent/php_txn.c

```diff
@@ -911,7 +911,8 @@ nr_status_t nr_php_txn_begin(const char* appnames,
     nr_php_txn_log_error_dt_on_tt_off();
   }
 
-#if ZEND_MODULE_API_NO >= ZEND_8_1_X_API_NO
+#if ZEND_MODULE_API_NO >= ZEND_8_1_X_API_NO \
```

Comment:
Is this functionality unique to 8.1 and not applicable to 8.0?

Reply:
Yes, it was special for 8.1